### ISSUE TITLE
Stripe ACH Support

### ISSUE THREAD

**Issue by tomschlick** [cartalyst/stripe#49]
Launched publicly today, looks to be the same interface for charging a card, would just need to add the interface to capture the bank details & verify.

https://stripe.com/blog/accept-ach-payments

https://stripe.com/docs/guides/ach

**Comment by brunogaspar** [comment]
Feel free to create a PR.

**Comment by tomschlick** [comment]
I may do that as I get closer to implementing it. Was just opening it up for discussion first. Sometimes projects don't agree with everything being implemented or someone is already working on it.

**Comment by brunogaspar** [comment]
Correct and i agree.

But by looking into their docs, i don't see any change to be required at the API level.

**Comment by tomschlick** [comment]
Probably the manual verification step (them depositing two amounts into the users account and the user confirming in your own system). To my knowledge there isn't that endpoint here yet...

```php
// verify the account
$bank_account->verify(array('amounts' => array(32, 45)), null);
```

https://stripe.com/docs/guides/ach#manually-collecting-and-verifying-bank-accounts

**Comment by brunogaspar** [comment]
That endpoint doens't seem to be listed on the API docs (at least i'm no seeing it) but we can easily add that.

**Comment by brunogaspar** [comment]
@tomschlick Have you tried this? Do we need to make any changes on the package?

**Comment by tomschlick** [comment]
Haven't had a chance to fully implement yet as some priorities shifted at work. Going to try to get it done in the next week or so.

The only changes I would see needing is the verify() method if it doesn't already exist here. Doesn't look like it's documented in their API spec but their php library implementation is here... 

https://github.com/stripe/stripe-php/blob/master/lib/BankAccount.php#L13

**Comment by brunogaspar** [comment]
Cool, not worries.

I'll have a look if i have time as well, but that method is not implemented on our end at the moment.

**Comment by brunogaspar** [comment]
Implemented this on 1a8df1aa8f8dac88a81abd6e2510dec92b142cbf.

The usage is very easy and similar to what we have :)

```php
$stripe->bankAccounts()->verify(:customerId, :bankAccountId, array :amounts);
```
